client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -448,10 +448,10 @@ if(WIN32 AND "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
             dbghelp)
 endif()
 
-if(NOT WIN32)
+if(CMAKE_SYSTEM_NAME STREQUAL "QNX")
     set(KAA_THIRDPARTY_LIBRARIES
             ${KAA_THIRDPARTY_LIBRARIES}
-            pthread)
+            socket)
```

Comment:
pthread was removed for all non-windows platforms. We should recheck if it's really needed.